README_PYPI.md

```diff
@@ -0,0 +1,277 @@
+- Provide both a command line mode and an interactive mode where
+  multiple pywbemcli commands can be executed within the context of a
+  WBEM server.
+- Use an integrated mock WBEM server to try out commands. The mock
+  server can be loaded with CIM objects defined in MOF files or via
+  Python scripts.
+
+# Documentation
+
+For the latest version of pywbemtools released on Pypi:
+
+- [Documentation](https://pywbemtools.readthedocs.io/en/stable/)
+- [Installation](https://pywbemtools.readthedocs.io/en/stable/introduction.html#installation)
+- [Change history](https://pywbemtools.readthedocs.io/en/stable/changes.html)
+
+# Quickstart
+
+All commands within pywbemcli show help with the `-help` or `-h` options:
+
+``` text
+$ pywbemcli --help
+. . .
+$ pywbemcli connection --help
+. . .
+$ pywbemcli connection save --help
+. . .
+```
+
+The following examples build on each other and show a typical sequence
+of exploration of a WBEM server. For simplicity, they all operate
+against the default namespace of the server, and use a persistent
+connection definition for the server:
+
+-   Add a persistent connection definition named `conn1` for the WBEM
+    server to be used, so that the subsequent commands can refer to it:
+
+    ``` text
+    $ pywbemcli -s https://localhost --no-verify -u user -p password connection save conn1
+    ```
+
+-   pywbemcli also supports mocked WBEM servers in memory, that are
+    preloaded with CIM objects defined in MOF files. Add a persistent
+    connection definition named `assoc1` to a mock server using one of
+    the MOF files provided in the repo:
+
+    ``` text
+    $ pywbemcli -m tests/unit/simple_assoc_mock_model.mof connection save assoc1
+    ```
+
+-   List the persistent connection definitions:
+
+    ``` text
+    $ pywbemcli connection list
+    WBEM server connections(brief): (#: default, *: current)
+    name    server             mock-server
+    ------  -----------------  --------------------------------------
+    assoc1                     tests/unit/simple_assoc_mock_model.mof
+    conn1   https://localhost
+    ```
+
+-   List the persistent connection definitions with full information:
+
+    ``` text
+    $ pywbemcli connection list --full
+    WBEM server connections(full): (#: default, *: current)
+    name    server             namespace    user      timeout  use_pull    verify    certfile    keyfile    mock-server
+    ------  -----------------  -----------  ------  ---------  ----------  --------  ----------  ---------  --------------------------------------
+    assoc1                     root/cimv2                  30              True                             tests/unit/simple_assoc_mock_model.mof
+    conn1   https://localhost  root/cimv2   user           30  True        False
+    ```
+
+-   Show the class tree, using the previously added connection
+    definition `assoc1`:
+
+    ``` text
+    $ pywbemcli -n assoc1 class tree
```

Comment:
add "where -n is a pywbemcli persistent connection name (see pywbemcli connection)

Reply:
We already clarify that partially in the sentence before the example. Added the `-n` option there:
```
Show the class tree, using the previously added WBEM server connection
definition `assoc1` (specified with the `-n` option):
```